hdfs3/core.py

```diff
@@ -672,6 +672,14 @@ def __iter__(self):
         """ Enables `for line in file:` usage """
         return self._genline()
 
+    def __next__(self):
+        """ Enables reading a file as a buffer in pandas """
+        return next(self._genline())
```

Comment:
Note that `readline()` is known to perform poorly (and seems somewhat odd for binary files), so I hope pandas does not really call this repeatedly. If it does, wrapping the file in `io.TextIOWrapper` is probably far better.